templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,21 @@
+<!-- templates/registration/password_reset_form.html -->
+{% extends 'base.html' %}
+
+{% comment %}
+  Password reset only sends emails to active accounts with a usable password.
+  https://www.reddit.com/r/django/comments/8picrc/send_mail_works_fine_but_resetpassword_email_does/e0bm6km/
+  https://github.com/django/django/blob/e0d0fc0b14ccade00b30a7fbdb1c720ddd83bb71/django/contrib/auth/forms.py#L257-L261
+{% endcomment %}
```

Comment:
Is it much effort to change this (usable password requirement)? It will eliminate the need to set passwords for new users